documentation/src/main/markdown/integrations/jenkinsplugin/pluginreleasenotes.md

```diff
@@ -1,5 +1,19 @@
 # Release Notes for Jenkins Plugin
 
+## **Version 10.0.0**
+
```

Comment:
@sig-cpotts Also, same note saying - It is essential to upgrade to 10.0.0 before detect.synopsys.com and sig-repo is decommissioned in Feb, 2025. Refer to https://community.blackduck.com/s/article/Black-Duck-Domain-Change-FAQ#SDInstru